BUILD.md

```diff
@@ -0,0 +1,39 @@
+Environment
+This documentation is intended for Ubuntu 20.4
+
+### Docker
+
+To begin with, you should install Docker. If you haven't installed it yet, please run the following command:
+
+`bash docker_install.sh`
```

Comment:
We do Not need to provide docker install scripe. We can just provide the official docker installationguide to reduce our documentation maintenance costs.
For example: https://docs.docker.com/engine/install/debian/